sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.Metrics.cs

```diff
@@ -33,7 +39,7 @@ public MonitoringDataPoint GetDataPoint()
                 StreamId = _streamId,
                 Timestamp = DateTime.UtcNow, // Represents timestamp sample was created
                 TransmissionTime = DateTime.UtcNow, // represents timestamp transmission was sent
-                IsWebApp = IsWebAppRunningInAzure(),
+                IsWebApp = IsWebAppRunningInAzure(), // TODO: THIS LOOKS LIKE A BUG. THIS METHOD RETURNS BOOL INDICATING IF IT SHOULD COLLECT PERF COUNTERS. AI SDK IS DOING THE SAME THING. NEED TO REVIEW THIS WITH SERVICE TEAM.
```

Comment:
I agree, swagger has a comment that says if app runs in Azure App Service. But in the actual method more checks are performed.